apps/api/src/services/providers/__tests__/multi-chain.provider.test.ts

```diff
@@ -111,44 +111,45 @@ describe("MultiChainProvider", () => {
       const duration = Date.now() - start;
 
       // Second call should be very quick (under 5ms) as it's using the cache
+      // TODO(stbrody): This is a bad way to check if the cache is being used. Make this check more targeted and less flaky.
```

Comment:
note: there are a few TODO comments. do we want to address these in this PR, or follow up?